### DISCUSSION TITLE
Osu has triggered an emergency shutdown

### DISCUSSION THREAD

**Discussion by Resoult-T** [ppy/osu#22223] [Q&A]
### Type

Crash to desktop

### Bug description

My pc immediatly shuts down when starting a beatmap after setting FPS limit to "unlimeted".
I can't say muche more about this problem. There are not realy logs to search for. 
Eventlog created an Aplication error osu!.exe -> atioglxx.dll. Thats everything. GPU and CPU temperatures where stable. 

GPU: Radeon RX 6950XT
CPU: Ryzen 5 5600X
RAM: 16GB 3200MHz

This was the case two times in a row. I did not want to try it out again.

### Version

b20230101.2

### Logs

[session.log](https://github.com/ppy/osu/files/10420497/session.log)
[update_success.log](https://github.com/ppy/osu/files/10420498/update_success.log)
[osu!auth.log](https://github.com/ppy/osu/files/10420499/osu.auth.log)
[runtime.log](https://github.com/ppy/osu/files/10420500/runtime.log)

**Comment by bdach** [comment]
This is a driver crash and not one we can do much about right now. This will be hopefully addressed once the default renderer is changed away from opengl. You can try updating your graphics cards drivers in the mean time.

Also please note that in general there is *no reason* for you to play with "unlimited". Please read https://github.com/ppy/osu/wiki/Latency-and-unlimited-frame-rates.

**Reply by Resoult-T** [reply]
I was trying because the latence was much lower than the other settings. (0.32ms)
With "recomended" settings it is at 1.4ms so also totaly fine.
I've created this issue because i've never experienced this.

Thanks for your quick answer!

**Reply by Loreos7** [reply]
You're talking about **frame time**. In lazer this thing doesn't affect input latency.
